pkgs/net-doc/net/scribblings/git-checkout.scrbl

```diff
@@ -47,7 +47,8 @@ to download the repository whose name on the server is
 @racket[repository] (normally ending in @filepath{.git}).  The tree
 within the repository that is identified by @racket[ref] (which can be
 a branch, tag, commit ID, or tree ID) is extracted to
-@racket[dest-dir], and the result id an ID corresponding to @racket[ref].
+@racket[dest-dir], and it returns a string containing a commit id corresponding 
```

Comment:
Nit: "ID" instead of "id" for consistency.

Reply:
Sure.